src/test/app/common/calculateInterest.ts

```diff
@@ -47,6 +47,15 @@ describe('calculateInterest', () => {
         expect(calculateInterest(100, interest, interestDate)).to.equal(0)
       })
     })
+
+    describe('interest type is STANDARD and', () => {
```

Comment:
And? No need to say that, the way test results are displayed does make having `and` at the end useful.